wiki/WebviewView.md

```diff
@@ -0,0 +1,36 @@
+Represents a extension-contributed view that's displayed in the bottom or side bar.
```

Comment:
could you please include also screenshot with precise selection about which part of editor is this wiki page about? I would like to keep it same as it was done for other page objects.

Reply:
I added an image to the wiki page

Reply:
thank you